packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -146,34 +149,46 @@ class FlutterDriver {
   /// [logCommunicationToFile] determines whether the command communication
   /// between the test and the app should be logged to `flutter_driver_commands.log`.
   ///
+  /// [isolateNumber] (optional) determines the specific isolate to connect to.
+  /// If this is left as zero, will connect to the first isolate found running
```

Comment:
Maybe use `null` as the default instead of zero in case zero is ever a valid number?

Reply:
Good point.  I'm still stuck in the mindset that lowercase types like `int` are primitives instead of objects.